api/src/main/java/org/pmiops/workbench/db/dao/RdrExportDao.java

```diff
@@ -6,15 +6,20 @@
 import org.pmiops.workbench.db.model.DbRdrExport;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
 
 public interface RdrExportDao extends CrudRepository<DbRdrExport, Long> {
 
-  @Query(
-      nativeQuery = true,
-      value =
-          "select u.user_id from user u LEFT JOIN rdr_export rdr on"
-              + " u.user_id = rdr.export_id and rdr.entity_type = 1 where "
-              + "u.last_modified_time > rdr.last_Export_date or rdr.export_id is null")
+  public final String workspaceIdsToExportQuery =
+      "select w.workspace_id from  workspace w LEFT JOIN "
+          + "rdr_export rdr on w.workspace_id = rdr.entity_id and rdr.entity_type = 2 where "
+          + "w.last_modified_time > rdr.last_export_date or rdr.entity_id IS NULL";
+  public final String unchangedWorkspaceIdsQuery =
```

Comment:
Why are we only backfilling unchanged workspace IDs? Is this an optimization or is this necessary for the system to function properly? Should comment somewhere